src/pcre2_jit_compile.c

```diff
@@ -12053,12 +12054,7 @@ while (*cc != OP_KETRPOS)
       add_jump(compiler, &emptymatch, CMP(SLJIT_EQUAL, TMP1, 0, STR_PTR, 0));
 
     if (!zero)
-      {
-      if (framesize < 0)
-        OP1(SLJIT_MOV, SLJIT_MEM1(STACK_TOP), STACK(stacksize - 1), SLJIT_IMM, 0);
-      else
-        OP1(SLJIT_MOV, SLJIT_MEM1(STACK_TOP), STACK(0), SLJIT_IMM, 0);
-      }
+      OP1(SLJIT_MOV, SLJIT_MEM1(STACK_TOP), STACK(0), SLJIT_IMM, 0);
```

Comment:
Would be good to know why the original code is there.

Reply:
It looks like a copy & paste mistake that is also 10 years old, dunno though, I was not the author, it was part of f0855fab5